crt_portal/api/views.py

```diff
@@ -571,6 +572,37 @@ def get(self, request) -> JsonResponse:
         return upload_file(attachment)
 
 
+class ResourceContactView(APIView):
+    """
+    API endpoint that enables adding contacts to resources
+
+
+    Example: api/resource-contact-action/
+    """
+    form_class = ResourceContactActions
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request) -> JsonResponse:
+        action = request.POST.get('action')
+        if action == 'removed':
+            try:
+                resource_contact_id = int(request.POST.get('contact_id'))
+                resource_contact = get_object_or_404(ResourceContact, pk=resource_contact_id)
+                resource_contact.delete()
+                return JsonResponse({'response': f'Contact was successfully {action}', 'id': resource_contact_id, 'type': 'success'})
+            except ValueError:
```

Comment:
where does this value error come from? consider putting this only over the line you expect to ValueError to avoid overselecting.